static/app/components/demo/demoHeader.tsx

```diff
@@ -59,15 +71,18 @@ const LogoSvg = styled(IconSentryFull)`
   color: ${p => p.theme.textColor};
 `;
 
-const GetStarted = styled(Button)`
-  background-color: #e1567c;
-  color: #fff;
-  box-shadow: 0 2px 0 rgb(54 45 89 / 10%);
-  border-color: transparent;
+const BaseButton = styled(Button)`
   border-radius: 2rem;
   text-transform: uppercase;
 `;
 
+const GetStarted = styled(BaseButton)`
+  border-color: transparent;
+  box-shadow: 0 2px 0 rgb(54 45 89 / 10%);
+  background-color: #e1567c;
+  color: #fff;
```

Comment:
Can we leave a comment here saying these colors are intentional to match marketing palette?